ansible/roles/sonic-common/tasks/main.yml

```diff
@@ -18,6 +24,14 @@
 - include: aptrepo.yml
   tags: repo
 
+# Dbus
+- name: Install dbus
+  become: true
+  apt: pkg=dbus
```

Comment:
Looks like we really don`t need it. Will remove in next commit. Thanks for pointing it out.